src/core/border_router/routing_manager.hpp

```diff
@@ -1533,14 +1537,16 @@ class RoutingManager : public InstanceLocator
         void EvaluateStateChange(State aOldState);
         void WithdrawPrefix(void);
         void StartStop(bool aStart);
+        void PauseResume(bool aPause);
 
         static const char *StateToString(State aState);
 
         using PrefixTimer   = TimerMilliIn<RoutingManager, &RoutingManager::HandlePdPrefixManagerTimer>;
         using StateCallback = Callback<PdCallback>;
 
         bool          mEnabled;
-        bool          mIsRunning;
+        bool          mIsStarted;
+        bool          mIsPaused;
```

Comment:
`mIsPaused` seems relevant to `mIsStarted` so maybe  add a comment to clarify there meanings.

Reply:
Comments added